src/environments/environment.ts

```diff
@@ -4,7 +4,8 @@
 
 
 export const environment = {
-    production: false,
+  production: false,
+  customer_url: 'http://localhost:8080/customers/',
     baseURL: 'http://localhost:8080'
```

Comment:
Please build customer URL with base url+your path. Thus we will not have to change it multiple times

Reply:
Okay, got it.